### PR TITLE
website: add blog post about trying out Podman Desktop in a GitHub Codespace

### LINKED ISSUE #751: add blogging capabilities on the Podman Desktop website
### Is your feature request related to a problem? Please describe

Apart from documentation, there should be a separate section on the website for blog posts

### Describe the solution you'd like

A separate section / page where all blog posts can be accessed

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Blog post about trying out Podman Desktop in a GitHub Codespace

### Screenshot/screencast of this PR

image

### What issues does this PR fix or reference?

N/A

### How to test this PR?

`yarn website:dev`

### CODE REVIEW

File: website/blog/2022-11-17-develop-podman-using-codespaces.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+description: Develop Podman Desktop using a DevContainer locally or using GitHub Codespaces.
+slug: develop-using-devcontainer
+authors: [benoitf]
+tags: [podman-desktop, devcontainer, codespaces]
+hide_table_of_contents: false
+---
+
+GitHub [announced last week](https://github.blog/changelog/2022-11-09-codespaces-for-free-and-pro-accounts/) that Codespaces is available for everyone and it includes free minutes.
+
+Let see how we can use a [Development Container](https://containers.dev/) having all the tools to build and run Podman Desktop. The development Container works locally using Visual Studio Code but in this blog post we will see how it works with a simple click on GitHub.
```
Comment (slemeur) [inline website/blog/2022-11-17-develop-podman-using-codespaces.md]:
```suggestion
Let see how we can use a [Development Container](https://containers.dev/) having all the tools to build and run Podman Desktop. The Development Container works locally using Visual Studio Code but in this blog post we will see how it works directly with a simple click from GitHub.
```

File: website/blog/2022-11-17-develop-podman-using-codespaces.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+The challenges are to run a desktop tool and running a container engine (Podman) inside a container and using not so many memory !
```
Comment (slemeur) [inline website/blog/2022-11-17-develop-podman-using-codespaces.md]:
```suggestion
The challenges are to run a desktop tool (Podman Desktop) and running a container engine (Podman) inside this Development Container without using too many memory !
```

File: website/blog/2022-11-17-develop-podman-using-codespaces.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+ENV _CONTAINERS_USERNS_CONFIGURED=""
+```
+
+Make sure Podman will create the socket in an expected directory:
+```
+# socket path for podman
+ENV XDG_RUNTIME_DIR=/run/user/1000
+```
+
+OK ! we have a custom Containerfile providing all the tools to build and run Podman Desktop, run Podman and run Electron.
```
Comment (slemeur) [inline website/blog/2022-11-17-develop-podman-using-codespaces.md]: and connect to it with novnc?

File: website/blog/2022-11-17-develop-podman-using-codespaces.md [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+```
+
+Some Visual Studio Code extensions are nice to use and we can add them
+```json
+  "extensions": ["svelte.svelte-vscode", "bradlc.vscode-tailwindcss"]
+```
+
+Then here is the tricky part, how to run our container allowing to run again inside the container some containers with podman.
+
+We specify the arguments to make it possible. It's possible to use `--privileged` flag but I prefer to list the subset of permissions.
```
Comment (slemeur) [inline website/blog/2022-11-17-develop-podman-using-codespaces.md:172]: optional explain why as it is not necessarily obvious to all

File: website/blog/2022-11-17-develop-podman-using-codespaces.md [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+
+It's also possible using the port widget to get on `3000` port by clicking on the world icon a preview of the website in another tab. Changing source code of the website will refresh the content of the window.
+
+Depending on the usecase, it's also possible to open documentation in the preview browser.
+
+![Edit website Codespace](img/develop-podman-using-codespaces/codespaces-edit-website.png)
+
+## Conclusion
+
+The DevContainer image for Podman Desktop is recent so it'll probably evolve over time by adding new capabilities but it allows you to build/run/experiment the tool or to contribute to the website.
```
Comment (slemeur) [inline website/blog/2022-11-17-develop-podman-using-codespaces.md]:
```suggestion
The DevContainer image for Podman Desktop is recent so it'll probably evolve over time by adding new capabilities but it allows you to easily build/run/experiment and **contribute** to the tool or the website.
```

**Review by slemeur — APPROVED** [review]
Some little tweaks, but this is awesome!

**Comment by benoitf** [comment]
I've applied suggested changes from Stevan + added extra details

**Review by cdrage — APPROVED** [review]

**Pull request merged**